src/test/java/com/artipie/asto/RemainingTest.java

```diff
@@ -32,6 +32,9 @@
 /**
  * Test case for {@link Remaining}.
  * @since 1.0
+ * @todo #196:30m This since tag is incorrect.
+ *  Replace version here and in @{link KeyTest} with
+ *  correct number.
```

Comment:
@andreoss sorry, but I think it does not deserve to be a separate task. Probably to change the version change or leave it as it is without adding `@todo`

Reply:
@genryxy Removed